### PR TITLE
Revert "USEID-915: Enable export per record (#1)"

### PR DESCRIPTION
**Pull request by pgurusinga**
This reverts commit 77b2e481606369b5311298bb6f77d7da71d60dec.

### CODE REVIEW

File: query_and_submit.py [hunk 1]
```diff
@@ -18,52 +18,22 @@
...
-        **content_metric
-      })
-
-  else:
-    condition_keyword = "and" if re.search("where", count_sql, re.IGNORECASE) else "where"
-    count_metrics[metric_name] = json.loads(subprocess.check_output(["psql", os.environ["DBURI"], "-Atc", f"{count_sql} {condition_keyword} \"{timestamp_col}\" between '{interval_start}' and '{interval_end}'"]))
-
-if count_metrics:
-  metrics.append({
+requests.post(
```
Comment (JulianRoesner) [inline query_and_submit.py]: I am wondering whether we want to keep the response check: 
```suggestion
response = requests.post(
```

File: query_and_submit.py [hunk 2]
```diff
@@ -18,52 +18,22 @@
...
+  },
   headers={
     "xc-token": metrics_token,
   }
 )
-
-if response.ok:
-  print("export done")
-else:
-  print("export failed")
```
Comment (JulianRoesner) [inline query_and_submit.py]: And then also keep this

**Review by JulianRoesner — APPROVED** [review]
LGTM. However, I would keep a some of the improvements. I think we also need to change how we use this job in our infrastructure first.

File: query_and_submit.py (hunk 1 above)
Comment (pgurusinga) [reply query_and_submit.py]: good idea 👏

**Pull request merged**